lib/src/view/analysis/analysis_share_screen.dart

```diff
@@ -75,6 +123,20 @@ class _EditPgnTagsForm extends ConsumerWidget {
                     (e) => showRatings || !_ratingHeaders.contains(e.key),
                   )
                       .mapIndexed((index, e) {
+                    if (!_controllers.containsKey(e.key)) {
+                      _controllers[e.key] =
```

Comment:
Not a big fan of doing side effects like that in the build method, but this is by far the easiest way here I reckon.

Reply:
I see. I guess the initialization belongs to initState. 😁